.gitlab-ci.yml

```diff
@@ -241,3 +241,15 @@ integration_mariadb:
     - name: mariadb
       alias: sqlserver
 
+php-cs-fixer:
+  image: 
+    name: cytopia/php-cs-fixer
+    entrypoint: [""]
+  script:
+    - /usr/bin/php-cs-fixer fix --dry-run --diff .
```

Comment:
If we run this with `--format=gitlab` and write it to a JSON + add an artifact like this:
```yaml
    artifacts:
        when: on_failure
        reports:
            codequality: code-quality.json
```

I think it at least shows up in a tab in the GitLab pipeline.

Reply:
I wonder if there is an github actions which can also do this, so the extra config to get it to GitHub is not needed. (I searched but didn't find the same feature yet)

Reply:
@nickygerritsen this is now implemented and can be found at: https://gitlab.com/DOMjudge/domjudge/-/pipelines/248383076/codequality_report

I think keeping this there and just always accepting the test would be fine? So have the report but dont fail and work more with the github actions which are now introduced.